packages/flutter/lib/src/widgets/scrollable.dart

```diff
@@ -147,6 +147,12 @@ abstract class ScrollableState<T extends Scrollable> extends State<T> {
     return _scrollBehavior;
   }
 
+  void resetScrollBehavior() {
+    setState(() {
+      _scrollBehavior = null;
+    });
+  }
```

Comment:
This pattern seems less than ideal...  Maybe the subclass should override `get scrollBehavior` to customize it entirely?